build.sbt

```
@@ -69,6 +69,7 @@ val betterMonadicForV = "0.3.1"
 // General Settings
 lazy val commonSettings = Seq(
   Test / parallelExecution := false,
+  resolvers += "Sonatype OSS Snapshots" at "https://s01.oss.sonatype.org/content/repositories/snapshots",
```

This will include all of them :)
```suggestion
  resolvers ++= Resolvers.sonatypeOssRepos("snapshots"),
```